Readme.md

```diff
@@ -1,21 +1,69 @@
-# Lombiq <add project name here>
+# Lombiq PowerShell Analyzers
 
 
 
 ## About
 
-Add a general overview of the project here. Keep or remove the OSOCE note below as necessary.
+Powershell static code analysis via [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) and [our configuration for it](PSScriptAnalyzerSettings.psd1).
 
 Do you want to quickly try out this project and see it in action? Check it out in our [Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) full Orchard Core solution and also see our other useful Orchard Core-related open-source projects!
 
 
 ## Documentation
 
-Add detailed documentation here. If it's a lot of content then create documentation pages under the *Docs* folder and link pages here.
+### Pre-requisites
 
+The *PSScriptAnalyzer* module must be installed. If it's not installed, then running the script in `pwsh` will output no results and running it in `powershell` will throw an exception. Follow the steps [here](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/overview?view=ps-modules#installing-psscriptanalyzer).
+
+Note that if you are usnig this in GitHub Actions, the common images (`windows-latest` and `ubuntu-latest`) already have it so you don't need to install anything.
+
+### Usage
```

Comment:
Can we also use it in an editor, to see squiggly lines as with .NET analyzers?

Reply:
I don't know. Don't they need to be running live for that to work?

Reply:
There is an extension for VSC which works great, but you need to set the PSScriptAnalyzerSettings.psd1 path in the settings, otherwise it only looks at the currently opened folder. There is no extension for Visual Studio. There is an extension for Rider, but the one for Code is better so that's what I mainly use for PS. (if you don't juggle multiple IDEs are you even a real programmer? 😁)

Added note on setting up the Code extension with our settings file in the readme. I don't think we can offer anything more in the scope of this issue.

Reply:
Sounds good.